File: C/marioh.c
#include <stdio.h>

void printDoublePyramid(int height)
{
    for (int i = 1; i <= height; i++)
    {
        for (int j = 0; j < height - i; j++)
        {
            printf(" ");
        }
        for (int k = 0; k < i; k++)
        {
            printf("#");
        }
        printf("  ");
        for (int k = 0; k < i; k++)
        {
            printf("#");
        }
        printf("\n");
    }
}

int main()
{
    int height = 8;
    printDoublePyramid(height);
    return 0;
}

File: Python/marioh.py
def print_double_pyramid(height):
    for i in range(1, height + 1):
        spaces = " " * (height - i)
        hashes = "#" * i
        print(spaces + hashes + "  " + hashes)


if __name__ == "__main__":
    height = 8
    print_double_pyramid(height)
